cmd/vfkit/main.go

```diff
@@ -59,6 +60,9 @@ func newBootloaderConfiguration(opts *cmdline.Options) (config.Bootloader, error
 }
 
 func newVMConfiguration(opts *cmdline.Options) (*config.VirtualMachine, error) {
+	runtime.LockOSThread()
+	defer runtime.UnlockOSThread()
```

Comment:
This did not seem required in my testing (latest macOS 13 on a m1 macbook), do you remember why you added this?

Reply:
This isn't required, I must have left it there accidentally